homeassistant/components/shelly/__init__.py

```diff
@@ -20,7 +20,7 @@
 
 from .const import DOMAIN
 
-PLATFORMS = ["binary_sensor", "light", "sensor", "switch"]
+PLATFORMS = ["binary_sensor", "light", "sensor", "switch", "cover"]
```

Comment:
It's nice to have sorted lists. :abcd: :wink:

Reply:
I was in the hope that **isort** would handle that as well

Reply:
No, isort only handles the imports.

Reply:
Sorry, will be more careful next time. Still learning all the code conventions in HA

Reply:
No worries. It's just a small thing.

Reply:
https://github.com/home-assistant/core/pull/39798